tests/test_load_data.py

```diff
@@ -289,3 +289,56 @@ def test_missing_file_handling():
     with pytest.raises(IOError):
         with ignore_exceptions_if(True, (ValueError, ArithmeticError)):
             rio_slurp('no-such-file.tiff')
+
+
+def test_native_load(tmpdir):
```

Comment:
Use the `tmp_path` fixture instead and you don't need to manually convert to a `pathlib.Path`

Reply:
Changed over to `tmp_path`

I put it into testutils on purpose for now, it's using old style IO driver and in general is just a stop-gap measure until proper native support is implemented with db support.

Reply:
Fair enough. I was thinking that somewhere storage related but specifically marked as /temporary/ and /user beware/, rather than a test utility. But it's really not worth the effort and potential confusion. :)